pyisic/_standards/ateco/ateco_to_nace2.py

```diff
@@ -0,0 +1,880 @@
+"""Concordance between ATECO and NACE2.
+
+"""
```

Comment:
Can we update this docstring to include reference to where the data came from?

Reply:
Added.